py/jupyterlite/src/jupyterlite/config.py

```diff
@@ -64,9 +64,10 @@ class LiteBuildConfig(LoggingConfigurable):
         config=True
     )
 
-    ignore_sys_prefix: bool = Bool(
-        False,
-        help="ignore lab components from sys.prefix, such as federated_extensions",
+    ignore_sys_prefix: _Tuple[str] = TypedTuple(
+        Unicode(),
+        help="list of components from sys.prefix to ignore."
+        "Values: ['federated_extensions', 'mathjax', 'translations']",
```

Comment:
Hm... this is probably in the same domain as `disable_addons`, and could be based on the entry_point key.

Reply:
also, perhaps it is perhaps a `Union(Boolean(), TypedTuple(Unicode()))`?

Reply:
The main motivation was to keep the existing `LiteBuildConfig` as the place for configuring the build.

https://github.com/jupyterlite/jupyterlite/pull/621/commits/05eb8aa221582192510310aadcdc12a629490972 takes one step towards moving that logic to the addons.

Maybe we could indeed keep the `ignore_sys_prefix` on `LiteBuildAddon` similar to `disable_addon`. And set the corresponding `ignore_sys_prefix` on each of the addons.